### PR TITLE
(deleted)

### PR DESCRIPTION
**Pull request by geo-y**

### CODE REVIEW

**Comment by hauke** [comment]
Please add a ipq40xx prefix to the patch and the commits

File: target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-cm520-79f.dtsi [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+&usb3_hs_phy {
+	status = "okay";
+};
+
+&usb2_hs_phy {
+	status = "okay";
+};
+
+&wifi0 {
+	status = "okay";
```
Comment (chunkeey) [inline target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-cm520-79f.dtsi:256]: This looks like an older version of the qcom-ipq4019-cm520-79f?

I.e. This was before
https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=821b6640994df32d20a087a01b7c368187a93c24 

and 

https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=cfc13c44595db591092859fc6adc71f1d8159c50

These commits added nvmem-cells and removed the userspace extraction.

Please update your source material.

**Comment by chunkeey** [comment]
Ok, this makes sense now. You went with the openwrt-21.02 branch. That's why that patches were missing.
Could you rebase your patches on top of the master branch please?

https://github.com/openwrt/openwrt/tree/master

**Comment by kody-wen** [comment]
> Ok, this makes sense now. You went with the openwrt-21.02 branch. That's why that patches were missing. Could you rebase your patches on top of the master branch please?
> 
> https://github.com/openwrt/openwrt/tree/master

Sure. I found that openwrt.git has serveral mirrors.
https://git.openwrt.org/openwrt/openwrt.git and https://github.com/openwrt/openwrt.git.
Where should I contribute? And which branch should I choose?

**Pull request closed without merging**